rotkehlchen/chain/ethereum/zerion.py

```diff
@@ -254,3 +256,68 @@ def handle_protocols(
             token_symbol=token_symbol,
             balance=Balance(amount=normalized_balance, usd_value=normalized_balance * usd_price),
         )
+
+
+# -- The below should be moved and integrated to the uniswap module
+
+class UniLPBalance(NamedTuple):
+    pool: DefiBalance
+    token0: DefiBalance
+    token1: DefiBalance
+
+
+def _decode_token(entry: Tuple) -> DefiBalance:
+    # TODO: This would need work to also calculate the usd value of the balance
+    decimals = entry[0][3]
+    return DefiBalance(
+        token_address=entry[0][0],
+        token_name=entry[0][1],
+        token_symbol=entry[0][2],
+        balance=Balance(
+            amount=token_normalized_value_decimals(entry[1], decimals),
+            usd_value=ZERO,
+        ),
+    )
+
+
+def _decode_result(data: Tuple) -> UniLPBalance:
+    pool_token = _decode_token(data[0])
+    token0 = _decode_token(data[1][0])
+    token1 = _decode_token(data[1][1])
+    return UniLPBalance(pool=pool_token, token0=token0, token1=token1)
+
+
+def uniswap_lp_token_balances(
```

Comment:
@vnavascues This is the function to query balances of uniswap lp tokens without the graph.